<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eventviewerui</class>
 <widget class="QMainWindow" name="eventviewerui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="usersFrame">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="liveCheckbox">
         <property name="text">
          <string>LIVE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="usersTree">
         <property name="indentation">
          <number>0</number>
         </property>
         <column>
          <property name="text">
           <string>Users</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="messagesFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0" colspan="3">
        <widget class="QTreeWidget" name="messagesTree">
         <property name="autoExpandDelay">
          <number>-1</number>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="headerMinimumSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>200</number>
         </attribute>
         <column>
          <property name="text">
           <string>User</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Message</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QDateTimeEdit" name="playbackTime">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="displayFormat">
          <string>M/d/yyyy h:mm:ssAP</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSlider" name="playbackProgress">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="playbackSpeed">
         <property name="suffix">
          <string>x</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
